vllm/model_executor/models/jais.py

```diff
@@ -243,19 +251,33 @@ def forward(
         position_ids: torch.Tensor,
         kv_caches: List[torch.Tensor],
         attn_metadata: AttentionMetadata,
+        intermediate_tensors: Optional[IntermediateTensors] = None,
+        inputs_embeds: Optional[torch.Tensor] = None,
     ) -> torch.Tensor:
-        inputs_embeds = self.wte(input_ids)
-        if self.wpe is not None:
-            position_embeds = self.wpe(position_ids)
-            hidden_states = inputs_embeds + position_embeds
+        if get_pp_group().is_first_rank:
+            if inputs_embeds is not None:
```

Comment:
With reference to the above comment about input_embeds, we don't need this conditional here.